addons/external_intercom/functions/fnc_parseClassnamesInput.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+
+/*
+  Name: TFAR_external_intercom_fnc_parseClassnamesInput
+
+  Author: Dorbedo, Arend
+    Parses an array of classnames, does a value check to ensure classnames
+    exist.
+
+  Arguments:
+    0: classnames <STRING>
+
+  Return Value:
+    parsed classnames <ARRAY>
+
+  Example:
+    ["[B_CrewHelmet, 51122, 52, 53, 4, 5, 56, 57, 58, 59, ""asd"", asd, ""88""]", 8, 87, 40] call TFAR_external_intercom_fnc_parseClassnamesInput;
+
+  Public: Yes
+*/
+params [
+    ["_valueString", "", [""]]
+];
+
+private _parsedValue = (_valueString splitString ",") apply {
+    if !(_x isEqualType "STRING") then {_x = format["%1", _x]};
+    [_x] call BIS_fnc_filterString;
```

Comment:
```suggestion
    [_x] call BIS_fnc_filterString
```
if you intend to return this value leave away semicolon on return values

Reply:
And does this apply to the end of file lines too? End of functions, where it outputs the function return?